components/Dialogs/src/MDCAlertController.m

```diff
@@ -212,6 +212,19 @@ - (void)setButtonTitleColor:(UIColor *)buttonColor {
   }
 }
 
+- (void)setCornerRadius:(CGFloat)cornerRadius {
+  _cornerRadius = cornerRadius;
+  if (self.alertView) {
```

Comment:
Please remove the `nil` check since `nil` safely receives all messages.

http://google.github.io/styleguide/objcguide.html#nil-checks